app/assets/stylesheets/building_map.scss

```diff
@@ -83,6 +83,26 @@
   content: '2';
 }
 
+ul {
```

Comment:
We should probably not style all `ul`s, at least that is usually not something one wants to do; instead, give the specific `ul` that should be styled like this a css `class` with the `class` attribute in the HTML and use `ul.my-class` to select it for styling here.